src/state/store.js

```diff
@@ -8,8 +9,21 @@ import userHomeReducer from 'home/homeSlice';
 import landscapeReducer from 'landscape/landscapeSlice';
 import sharedDataReducer from 'sharedData/sharedDataSlice';
 
+const handleAbortMiddleware = store => next => action => {
```

Comment:
question: what is this used for?

Reply:
@paulschreiber This is used to indetify a new type of request execution result when a request is aborted, so that we can handle that and avoid showing the user the result of request no longer relevant

Reply:
Can you tell me more? What changed about React 18 to require this?

Reply:
@paulschreiber This is also related to the double mounting of components that created the need for the `useFetchData` hook, so what react 18 is doing in development mode is mount and umount the component, that is when (for some route components) the calls to the API are triggered more than one time, and in the middle of that mount -> unmount -> mount there is an aborted request to the api that needs to be handled